src/vm/docs/mod.rs

```diff
@@ -1364,6 +1368,23 @@ The `id-header-hash` is the block identifier value that must be used as input to
 "
 };
 
+const GET_BURN_BLOCK_INFO_API: SpecialAPI = SpecialAPI {
+    input_type: "BlockInfoPropertyName, BlockHeightInt",
```

Comment:
Should we name the second arg `BlockHeightUint` instead?

Reply:
I copy/pasted this from "get-block-info".

You could argue that the "Height" implies that it is unsigned and so "BlockHeightUInt" is redundant.

Reply:
I see your point. 

Looking more broadly at this file though, it seems that `input_type` often just uses the actual clarity types, leaving the more descriptive names to the `signature` field. I think I would prefer that convention here.

Reply:
I was actually copying `get-block-info`.. should I just change both?

Reply:
I think that would be good, it would make the documentation more consistent overall

Reply:
done